Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic.py

```diff
@@ -0,0 +1,321 @@
+# --seedlist path\to\seedlist1.seed --seedlist path\to\seedlist2.seed <etc..> 
+
+# The following parameters are optional, but may be helpful:
+#   seedlist: an O3DE seedlist for describing what assets of the project should be bundled
+#   gem-folder-name-to-copy: A Gem Folder from <ProjectRoot>/Gems that should be copied to the output location of the exported project
+#                               Files found in these folders will go into <output-path>/Gems/GemFolderName
+#                               Variants for game/server launcher only are game-gem-folder-name-to-copy/server-gem-folder-name-to-copy resp.
+#   project-file-pattern-to-copy: Any regex pattern starting from <ProjectRoot> that indicates what files should be copied to output location.
+#                                   Files found in these patterns will go into <output-path>
+#                                   Variants for game/server launcher only are game-file-pattern-to-copy/server-file-pattern-to-copy resp.
+
+# For more information on the available parameters for this script, check the parse_arguments function defined at the bottom of the file
+
+def  build_non_monolithic_code_modules( project_path: pathlib.Path,
+                                        non_mono_build_path: pathlib.Path):
+    #build the pre_requisite tools if needed
+    os.makedirs(non_mono_build_path, exist_ok=True)
+
+    process_command(["cmake", "-B", non_mono_build_path, "-S", project_path])
+
+    process_command(["cmake", "--build", non_mono_build_path, "--target", "AssetProcessorBatch", "AssetBundlerBatch", "--config", "profile", "--", "/m"])
+
+def build_monolithic_code_modules( project_path: pathlib.Path,
+                        project_name: str,
+                        build_config: str,
+                        mono_build_path: pathlib.Path,
+                        should_build_game_launcher: bool,
+                        should_build_server_launcher: bool,
+                        should_build_unified_launcher: bool):
+    process_command(["cmake", "-B", mono_build_path, "-S", project_path, "-DLY_MONOLITHIC_GAME=1", "-DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0"])
+
+    os.makedirs(mono_build_path, exist_ok=True)
+    if should_build_server_launcher:
+
+        process_command(["cmake", "--build", mono_build_path, "--target", f"{project_name}.ServerLauncher", "--config", build_config, "--", "/m"])
+
+    if should_build_game_launcher:
+        process_command(["cmake", "--build", mono_build_path, "--target", f"{project_name}.GameLauncher", "--config", build_config, "--", "/m"])
+    
+    if should_build_unified_launcher:
+        process_command(["cmake", "--build", mono_build_path, "--target", f"{project_name}.UnifiedLauncher", "--config", build_config, "--", "/m"])
+
+
+def kill_existing_processes(project_name: str):
+    o3de_process_names = ['o3de', 'editor', 'assetprocessor', f'{project_name.lower()}.serverlauncher', f'{project_name.lower()}.gamelauncher' ]
```

Comment:
Killing the `o3de`, `editor` and `assetprocessor` applications aren't relevant, since none of those applications are being built.

Reply:
The second issue is what I ran into during test builds. Locked files in the project folder b/c I accidentally left o3de/project manager running during builds halted the build process. By checking for this first, it doesn't leave the build process in an unfinished state (either it's able to start, or not at all).

Reply:
Instead of killing those processes, maybe the script should error out saying that you must close those processes before continuing